qtpy/Qt3DAnimation.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright © 2009- The Spyder Development Team
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+"""Provides Qt3DAnimation classes and functions."""
+
+# Local imports
+from . import PYQT5, PYSIDE2, PythonQtError
+
+if PYQT5:
+    from PyQt5.Qt3DAnimation import *
+elif PYSIDE2:
+    # https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-1026
+    import PySide2.Qt3DAnimation as __temp
+    import inspect
+    for __name in inspect.getmembers(__temp.Qt3DAnimation):
+        globals()[__name[0]] = __name[1]
```

Comment:
This looks very odd, although I understand it's a bug in PySide2. Would you be willing to change it when the bug is fixed upstream?

Reply:
Yes, of course, I opened the issue so I'll get notified if anything moves upstream.

Reply:
Ok, that's all I wanted to know. Thanks!